src/util/chains.ts

```diff
@@ -8,6 +10,8 @@ export enum ChainId {
   OPTIMISTIC_KOVAN = 69,
   ARBITRUM_ONE = 42161,
   ARBITRUM_RINKEBY = 421611,
+  POLYGON = 137,
+  POLYGON_MUMBAI = 80001,
 }
 
 export const ID_TO_CHAIN_ID = (id: number): ChainId => {
```

Comment:
this feels like something that should be a typeguard function, switch feels unnecessary
https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards